### PR TITLE
[IMP] web: remove bottom black border from trash icon in editable list

### PR DESCRIPTION
**Pull request by hmo-odoo**

### CODE REVIEW

File: addons/web/static/src/scss/list_view.scss [hunk 1]
```diff
@@ -91,6 +91,9 @@
         display: table-cell;
         cursor: pointer;
     }
+    .o_data_row.o_selected_row > .o_list_record_remove {
+            border-bottom: 1px solid $table-border-color;
```
Comment (aab-odoo) [inline addons/web/static/src/scss/list_view.scss]: indentation

**Comment by aab-odoo** [comment]
This is a fix, not an imp

File: addons/web/static/src/scss/list_view.scss (hunk 1 above)
Comment (hmo-odoo) [reply addons/web/static/src/scss/list_view.scss]: also, need to move inside ```&.o_editable_list```

**Comment by hmo-odoo** [comment]
oh..issue should be fixed in the enterprise only. community branch does not have an issue like this.

**Pull request closed without merging**